rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -105,6 +99,24 @@ object Runtime {
       9L -> SystemProcesses.secp256k1Verify(space, dispatcher)
     )
 
+    lazy val replayDispatchTable: Map[Ref, Seq[ListChannelWithRandom] => Task[Unit]] = Map(
+      0L -> SystemProcesses.stdout,
+      1L -> SystemProcesses.stdoutAck(replaySpace, dispatcher),
```

Comment:
should this use `dispatcher` or `replayDispatcher`?

Reply:
Oops.

Reply:
Nice catch; otherwise we might have gotten a surprise when we implemented unifications